pynotify/handlers.py

```diff
@@ -23,20 +23,21 @@ def __new__(cls, name, bases, attrs):
                 # delete the abstract attribute so it is not inherited by child classes
                 delattr(new_class.Meta, 'abstract')
             else:
-                if not getattr(new_class, 'Meta', None) or not getattr(new_class.Meta, 'signal', None):
-                    raise ImproperlyConfigured('No signal defined in {}\'s Meta.'.format(name))
+                if not getattr(new_class, 'Meta', None) or not hasattr(new_class.Meta, 'signal'):
```

Comment:
in django is used specific method get_options to get Meta. If no Meta is found is used default Meta class. It is much prettier and no hasattr is not required in th code.

Reply:
I was unable to find such method, where is it located? Anyway, please note, that handlers are not models, I think it is not a good idea to use any model-specific Django logic here.

Reply:
I don't want from you to use django logic. I want use the same principle. Because this is exactly the same situation. Code with hasattr, getattr is ugly and will increase how Meta class will grow. 

But you can leave it now. Probably you will find it later.